homeassistant/components/sensor/coinmarketcap.py

```diff
@@ -0,0 +1,128 @@
+"""
+Details about crypto currencies from CoinMarketCap.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.coinmarketcap/
+"""
+import logging
+from datetime import timedelta
+import json
+from urllib.error import HTTPError
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['coinmarketcap==2.0.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_24H_VOLUME_USD = '24h_volume_usd'
+ATTR_AVAILABLE_SUPPLY = 'available_supply'
+ATTR_MARKET_CAP = 'market_cap_usd'
+ATTR_NAME = 'name'
+ATTR_PERCENT_CHANGE_24H = 'percent_change_24h'
+ATTR_PERCENT_CHANGE_7D = 'percent_change_7d'
+ATTR_PRICE = 'price_usd'
+ATTR_SYMBOL = 'symbol'
+ATTR_TOTAL_SUPPLY = 'total_supply'
+
+CONF_CURRENCY = 'currency'
+
+DEFAULT_CURRENCY = 'bitcoin'
+
+ICON = 'mdi:currency-usd'
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=15)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_CURRENCY, default=DEFAULT_CURRENCY): cv.string,
+})
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the CoinMarketCap sensor."""
+    from coinmarketcap import Market
+    currency = config.get(CONF_CURRENCY)
+
+    try:
+        Market().ticker(currency)
+    except HTTPError:
+        _LOGGER.error('Currency "%s" is not available. Using "bitcoin"',
```

Comment:
I will change that. 

Note for me: Fix at least the `bitcoin` sensor. Use `warning` instead of `error`.